sdk/mx.sdk/mx_sdk_vm_impl.py

```diff
@@ -125,7 +125,7 @@ def svm_experimental_options(experimental_options):
     dir_name='polyglot',
     launcher_configs=[mx_sdk_vm.LauncherConfig(
         destination='bin/<exe:polyglot>',
-        jar_distributions=['sdk:LAUNCHER_COMMON'],
+        jar_distributions=['sdk:LAUNCHER_COMMON', 'truffle:TRUFFLE_API'],
```

Comment:
What is this change for? A launcher should never directly depend on truffle. Could you please revert this?